### DISCUSSION TITLE
Working with different human reference genomes

### DISCUSSION THREAD

**Discussion by gerpervaz** [zhengxwen/HIBAG#17] [Q&A]
I have SNP dataset in hg38 and I want to use HIBAG with prefit classifiers of illumina multiplatform such as the ImmunoChip-Broad-HLARES-HLA4-hg19 included in the training file. Do I need to convert  my dataset in hg19 or it is enough to declare it is in hg38 with the assembly option when converting my data from plink to genotype object data? I mean, is it enough to code this?
`hlaBED2Geno(study.bed, study.fam, study.bim, assembly="hg38")`

Thanks

**Comment by zhengxwen** [comment]
You should convert your data to hg19, since the pre-fit models were built on hg19.
